Network/network-exchange-clc-endpoint-guide.md

```diff
@@ -25,6 +27,8 @@ To create a dedicated virtual routing table (Dedicated VR), the VR will need to
 ### Notes
 
 * The Network Exchange fabric leverages the economics of shared networking while logically isolating network traffic between Exchanges, even within the same End User account. Dedicated access to CenturyLink Cloud is not supported with Network Exchange.
-* Currently, an End User may only add the CenturyLink Cloud endpoint(s) present in the same metropolitan area as the serving instance of Network Exchange. Where more than one CLC endpoint is served, all may be included in a given Exchange. See the *Network Exchange Availability Matrix and Configuration Guide* for more information.
+* Currently, an End User may only add the CenturyLink Cloud endpoint(s) present in the same metropolitan area as the serving instance of Network Exchange. Where more than one CLC endpoint is served, all may be included in a given Exchange. See the [Network Exchange Availability Matrix and Configuration Guide](https://www.ctl.io/knowledge-base/network/network-exchange-connectivity-matrix-configuration-guide/) for available endpoints per metro area.
 * A CLC service ticket is not required to add or delete a CenturyLink Cloud endpoint.
-* For additional information regarding the Network Exchange architecture and use cases, please consult the *Network Exchange Architecture and Technical Guide*.
+* For additional information regarding the Network Exchange architecture and use cases, please consult the [Network Exchange Architecture and Technical Guide](https://cloudhelp.ctl.io/knowledge-base/network-exchange-technical-guide/)
```

Comment:
I don't think we should be linking to the internal cloud help site in a Public KB.